app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -65,7 +65,7 @@
       android:textSize="16sp"
       android:visibility="gone"
       app:layout_constrainedWidth="true"
-      app:layout_constraintEnd_toEndOf="@{viewModel.isCorrectAnswer ? @id/answer_tick_guideline : ConstraintSet.PARENT_ID}"
+      app:layout_constraintEnd_toEndOf="@{viewModel.isCorrectAnswer &amp;&amp; !viewModel.hasConversationView ? @id/answer_tick_guideline : ConstraintSet.PARENT_ID}"
```

Comment:
This is fixing, but this issue is happening only for one answer and not for others?

Reply:
I am unable to understand what do you mean here?

As per the condition it is changing the end constraint for items which are correct and are in question player mode, that's because in question player mode right of correct answer we show a tick mark too but in exploration we do not show it.

Reply:
Got it from the meeting discussion.